clone_linkedin/post/views.py

```diff
@@ -56,6 +60,37 @@ def update(self, request, pk=None):
     def destroy(self, request, pk=None):
         self.get_object().delete()
         return Response()
+        
+    # GET, POST, PUT, DELETE /posts/:id/reaction/
+    @action(detail=True, methods=['GET', 'POST', 'PUT', 'DELETE'], url_path='reaction')
+    def reaction(self, request, pk=None):
+        post = self.get_object()
+        data = request.data.copy()
+        if self.request.method == 'GET':
+            reaction = post.postReactions.all()
+            return Response(self.get_serializer(reaction, many=True).data)
+        elif self.request.method == 'POST':
+            user = User.objects.get(id=1)          # Needs to be fixed after login implemented.
+            if self.get_object().postReactions.filter(user=user).exists():
+                return Response({"error": "Reaction already made."}, status=status.HTTP_400_BAD_REQUEST)
+            data['user_id'] = 1
+            data['post_id'] = post.id 
+            serializer = self.get_serializer(data=data)
+            serializer.is_valid(raise_exception=True)
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        elif self.request.method == 'PUT':
+            user = User.objects.get(id=1)          # Needs to be fixed after login implemented.
+            reaction = get_object_or_404(post.postReactions, user=user)
+            serializer = self.get_serializer(reaction, data=data, partial=True)
+            serializer.is_valid(raise_exception=True)
+            serializer.update(reaction, serializer.validated_data)
+            return Response(serializer.data)     
+        elif self.request.method == 'DELETE':
+            user = User.objects.get(id=1)       # Needs to be fixed after login implemented.
+            reaction = self.get_object().postReactions.get(user=user)     # Error if multiple reactions by the user exist
```

Comment:
여기도 `PUT`처럼 `get_object_or_404`로 하는게 좋을 것 같아요
~~지난번 review 땐 안 보였던게 왜 지금은 보일까요...~~
```suggestion
            reaction = get_object_or_404(self.get_object().postReactions, user=user)     # Error if multiple reactions by the user exist
```